src/main/java/teammates/logic/core/Emails.java

```diff
@@ -806,23 +805,24 @@ public MimeMessage sendErrorReport(HttpServletRequest req, Throwable error) {
             forceSendEmailThroughGaeWithoutLogging(email);
             log.severe("Sent crash report: " + Emails.getEmailInfo(email));
         } catch (Exception e) {
-            log.severe("Error in sending crash report: "
-                    + (email == null ? "" : email.toString()));
+            logErrorInSendingItem("crash report", email, e);
         }
     
         return email;
     }
 
     public MimeMessage sendLogReport(MimeMessage message) {
-        MimeMessage email = null;
         try {
             forceSendEmailThroughGaeWithoutLogging(message);
         } catch (Exception e) {
-            log.severe("Error in sending log report: "
-                    + (email == null ? "" : email.toString()));
+            logErrorInSendingItem("log report", message, e);
         }
+        return message;
+    }
     
-        return email;
+    private static void logErrorInSendingItem(String itemType, MimeMessage message, Exception e) {
```

Comment:
The `severe` part is important. The purpose of this logging is to let it be picked up by periodic severe log alerts. Either add a comment or rename to mention severe.